ensureconda/cli.py

```diff
@@ -20,7 +31,21 @@
     help="Search for conda.exe / conda-standalone, install if not present",
 )
 @click.option("--no-install", is_flag=True)
-def ensureconda_cli(mamba, micromamba, conda, conda_exe, no_install):
+@click.option(
+    "--min-conda-version", default=LooseVersion("4.8.2"), type=_as_loose_version
```

Comment:
Thanks for making it configurable instead of just hard-coding a minimum. While the latter would be easier to implement and cover 99% of the cases. I do have a 1% of those cases to maintain.